lang/main.json

```diff
@@ -470,16 +466,8 @@
     },
     "invite": {
         "addPassword": "Add password",
```

Comment:
Shouldn't we move the invite.addPassword to the "info" section now, because it's only used in the InfoDialog right now?